utils/products.ts

```diff
@@ -3,7 +3,11 @@
 
 import {ProductComponent} from '../types/store/plugins';
 
-export const getCurrentProductId = (products: ProductComponent[], pathname: string): string | null => {
+export const getCurrentProductId = (products: ProductComponent[] | undefined, pathname: string): string | null => {
+    if (pathname.startsWith('/people')) {
```

Comment:
This will get better semantics with `matchPath`-based checks in https://github.com/mattermost/mattermost-webapp/pull/11019/files#diff-306df37733404f27ab6dedef74b04b3ab268fb9fa9c1eb5ab7b063af88dff177R17